src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -316,6 +316,43 @@ internal static SemanticVersion PSV6Version
     }
 
     /// <summary>
+    /// Represents an implementation of '$PSVersionTable' variable.
+    /// The implementation contains ordered 'Keys' and 'GetEnumerator' to get user-friendly output.
+    /// </summary>
+    public sealed class PSVersionHashTable : Hashtable, IEnumerable
```

Comment:
> public class Hashtable : IDictionary, ICollection, IEnumerable, ...

Hashtable implements IEnumerable, so do you still need to specify `IEnumerable` here?